### DISCUSSION TITLE
Digests ?

### DISCUSSION THREAD

**Discussion by johnhenley** [DNNCommunity/Dnn.CommunityForums#129] [Q&A]
In reviewing the code changes required to store DateTimes as UTC and send emails to subscribers with their local datetime, I've stumbled across old code that is related to daily/weekly digests. It seems these were part of the old "Enterprise" version of Active Forums and have never been updated. In fact, they still reference "NTForums" tables :) Does anyone have interest in reviving these?

**Comment by WillStrohl** [comment]
Oooo!  That actually sounds like a great idea.  @Timo-Breumelhof What do you think?  😁

**Reply by Timo-Breumelhof** [reply]
agreed
